database/seeds/CamelSeeder.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+use Illuminate\Database\Seeder;
+
+class CamelSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        DB::table('shepherds')->insert([
```

Comment:
You could use the models here.

Reply:
Or much better, please create a factory. Also, if you always want to have a Vendég sheperd and a farmer, add them through migration.